[All Hosts] (yo office) new overview of the Yeoman generator tool (#3299)
OfficeDev/office-js-docs-pr: docs/testing/troubleshoot-manifest.md

![Animated GIF that shows the Yo Office validator being run at the command line and generating results that show Validation Passed.](../images/yo-office-validator.gif)

> [!NOTE]

> > To have access to this functionality, your add-in project must have been created by using [Yeoman generator for Office Add-ins](../develop/yeoman-generator-overview.md) version 1.1.17 or later.

Reviewer: ```suggestion
> To access this functionality, your add-in project must be created using the [Yeoman generator for Office Add-ins](../develop/yeoman-generator-overview.md) version 1.1.17 or later.
```
